.github/workflows/build.yml

```diff
@@ -37,15 +37,15 @@ jobs:
 
   build-macos:
     name: build in native macOS
-    runs-on: macos-10.15
+    runs-on: macos-latest
```

Comment:
should we pin this to latest ver instead of latest? 
worried about unexpected changes in future

Reply:
We do not actually build directly from from this branch. We have a Charles/arm64 branch we use to create our shared releases.